docs/installation.md

```diff
@@ -0,0 +1,23 @@
+# Installation Guide
+
+## Requirement
+
+This SDK requires JDK 1.6+. We use [okhttp](https://github.com/square/okhttp) and [org.json](https://github.com/stleary/JSON-java) to manage project dependences.
```

Comment:
It doesn't make sense, okhttp is an HTTP+HTTP/2 client , how to use it to manage project dependences ?

> We use [org.json](https://github.com/stleary/JSON-java) to manage project dependences.

looks better.

dependences will be described in org.json, and we don't need to repeat them in docs.